packages/lib/src/vis/line/DataCurve.tsx

```diff
@@ -29,6 +32,7 @@ interface Props {
   color: string;
   curveType?: CurveType;
   visible?: boolean;
+  onLineClick?: (index: number, evt: ThreeEvent<MouseEvent>) => void;
```

Comment:
I do not propagate the handler prop to `LineVis` as the only usecase we have deals with `DataCurve`